README.md

```diff
@@ -13,3 +13,7 @@ or
 
 yarn add @reactui-org/core
 ```
+
+## License
+
+[MIT](https://choosealicense.com/licenses/mit/)
```

Comment:
@Rishav1707 could we reference the license that exists in the project directories. you may find it [here](https://github.com/OSCA-Kampala-Chapter/react-ui/blob/main/LICENSE)

Reply:
```suggestion
[MIT](https://github.com/OSCA-Kampala-Chapter/react-ui/blob/main/LICENSE)
```